codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttDecoder.java

```diff
@@ -445,6 +438,21 @@ private static MqttFixedHeader decodeFixedHeader(ByteBuf buffer) {
         return new Result<String>(s, numberOfBytesConsumed);
     }
 
+    private static Result<byte[]> decodeByteArray(ByteBuf buffer, int minBytes, int maxBytes) {
+        final Result<Integer> decodedSize = decodeMsbLsb(buffer);
+        int size = decodedSize.value;
+        int numberOfBytesConsumed = decodedSize.numberOfBytesConsumed;
+        if (size < minBytes || size > maxBytes) {
```

Comment:
can we remove `minBytes` and `maxBytes` parameters? `maxBytes` is always `MAX_VALUE` and `minBytes` is always `0`. can we instead just throw an exception (or return early) if `size < 0`?